manifests/cluster-scope/kustomization.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
I prefer not to change the folder. Can we reuse `cluster-scope-resources`? Some projects (flyte, kubeflow) start to integrate with current repo and this may break their integration.